frontera/contrib/messagebus/zeromq/broker.py

```diff
@@ -48,6 +48,14 @@ def __init__(self, hostname, base_port):
         db_in_s = self.ctx.socket(zmq.XPUB)
         db_out_s = self.ctx.socket(zmq.XSUB)
 
+        if socket_config.is_ipv6:
+            spiders_in_s.ipv6 = True
```

Comment:
I also could not find it in the docs. I found it [here](https://github.com/zeromq/libzmq/issues/853).